x-pack/plugins/lists/server/services/exception_lists/utils/errors.ts

```diff
@@ -0,0 +1,18 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ErrorWithStatusCode } from '../../../error_with_status_code';
+
+export class DataValidationError extends ErrorWithStatusCode {
```

Comment:
I was thinking about adding support for error in the API error handling so that `reason` property can be returned in the API response. 

also, I may export this (along with `ErrorWithStatusCode`) from the Plugin so that other plugins that register extension points can also use them and thus be able to control the API's response code along with provide context specific error messages.

Thoughts on that?